.storybook/themes/news.scss

```diff
@@ -502,6 +502,18 @@
 					font-size: var(--global-font-size-13),
 					font-weight: var(--global-font-weight-7),
 				),
+				overline: (
```

Comment:
We don't need any of these values as they are all the same as the component values set above

Reply:
Kept the declaration for clarity but removed the values.  Did the same for the feature pack PR